reference/5-api-publisher/ConfigurationStore/Sql-Server.md

```diff
@@ -0,0 +1,233 @@
+# Configuration Store for SQL Server
+
+Enables management of individual connection settings with encryption support for securely storing keys and secrets using built-in support for symmetric key encryption in a SQL Server database.
+
+## Create the Ed-Fi API Publisher Configuration Database
+
+The following script template provides you with the information needed to create the `EdFi_API_Publisher_Configuration` database. Alternatively, you can perform this step manually using SSMS or through some other means and proceed with the next step.
+
```

Comment:
I'm curious, why are all of these SQL scripts in markdown files instead of being `.sql` files in the API Publisher repository?

Reply:
because they are being used as links in the "API-Connection-Management.md" file, now it could be changed to point to the ApiPublisher repository url.